docs/learn/learn-staking-miner.md

```diff
@@ -0,0 +1,79 @@
+---
+id: learn-staking-miner
+title: Staking Miner
+sidebar_label: Staking Miner
+description: Learn about Phragmén optimization on-chain via staking miners
+keywords: [staking, npos, miner, phragmén]
+slug: ../learn-staking-miner
+---
+
+:::caution
+The staking-miner code is experimental and it is still in development phase. Use is at your own discression, some funds might be lost.
+:::
+
+Staking miner is a passive validator functionality that runs when a new set of validators need to be elected for NPoS. This is a computationally intense process, hence the mining. Staking miners compete with each other to produce election solutions which consist of a validator set, stake distribution across that set, and a score indicating how optimal the solution is. Stakign miners run the sequential Phragmén algortihm to produce results, and the result is then sent as a transaction to the relay chain via a normal signed extrinsic. The transaction requires a bond, and a transaction fee. The best solution is rewarded
```

Comment:
"Staking miner is a passive validator functionality that runs" - this sentence might need rephrasing